<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>621</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="3" column="0" rowspan="6">
     <widget class="QScrollArea" name="ligandImageArea">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="ligandSearchText"/>
    </item>
    <item row="5" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="ligandSearchButton">
      <property name="text">
       <string>Search</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="resetNormalAAButton">
      <property name="text">
       <string>Reset to Standard AA</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="ligandTotalLabel">
      <property name="text">
       <string>Total: 20</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="searchOperationBox">
      <item>
       <property name="text">
        <string>name-exact</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>name-substring</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>name-close</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="pageBox">
      <property name="suffix">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="aminoAcidComboBox">
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="pageLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Page: 1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="browseSimilarButton">
      <property name="text">
       <string>Browse By Similarity</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="browseOperationBox">
      <item>
       <property name="text">
        <string>smiles</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>fpsmi</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QProgressBar" name="ligExpoProgressBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
